setup.py

```diff
@@ -34,20 +58,31 @@
         'License :: OSI Approved :: BSD License',
         'Programming Language :: Python',
         'Programming Language :: Python :: 3',
+        'Programming Language :: Python :: 3.6',
+        'Programming Language :: Python :: 3.7',
+        'Programming Language :: Python :: 3.8',
+        'Programming Language :: Python :: 3.9',
+        'Framework :: Jupyter',
     ],
     install_requires=[
-        'notebook'
+        'notebook',
```

Comment:
does not, in fact, require `notebook`. and further, would probably add a `jupyterlab >=3,<4` as we don't want to support 2.x (README should have at least a link to the previous version's readme)

Reply:
Yep. And updated.